simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/configuration/SimulatorConfiguration.java

```diff
@@ -42,6 +46,24 @@
         private int connectors;
     }
 
+    @Data
+    @Builder
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class MeterValuesConfiguration {
+        /**
+         * How often send meter values in milliseconds
+         */
+        @Builder.Default
+        private long meterValuesIntervalMs = DEFAULT_METER_VALUES_INTERVAL;
```

Comment:
Let's rename to 'sendMeterValuesIntervalSec'. And please mention that zero means never